microsoft-edge/progressive-web-apps-chromium/how-to/sidebar.md

```diff
@@ -0,0 +1,233 @@
+
+While building your app, you can test if the app's layout is usable when narrow, and responsive, by using the **Device Emulation** tool in Microsoft Edge DevTools. To learn more, see [Verify that the webpage layout is usable when narrow](../../devtools-guide-chromium/accessibility/narrow.md).
+
+If your app's layout can't support the 376 pixels minimum width, you can define your preferred width by using the `preferred_width` property in your web app manifest. For example:
+
+```json
+{
+  "name": "PWAmp music player",
+  "lang": "en-US",
+  "start_url": "/",
+  "short_name": "PWAmp",
+  "description": "A skinable music player app to play your favorite mp3 files",
+  "display": "standalone",
+  "icons": [
+    {
+      "src": "./favicon-256.png",
+      "sizes": "256x256"
+    }
+  ],
+  "edge_side_panel": {
+    "preferred_width": 480
+  }
+}
+```
+
+When you define a preferred width in your app's manifest, the following happens:
+
+* When your app is opened in the sidebar, the sidebar is automatically resized to your preferred width.
+* Users can still resize the sidebar to make it larger than your preferred width, but not smaller.
+
+
+<!-- ====================================================================== -->
+## Build a sidebar-only app
+
+One of the most important benefits of building an app as a PWA is that, from one code base, your app adapts to all devices and operating systems, whatever their capabilities and screen sizes.
+
+By using the `edge_side_panel` manifest member, you can also make your app installable as a standalone app, or pinned to the sidebar in Microsoft Edge.
+
+However, if you prefer to build an app that can only be pinned to the sidebar but not installed as a standalone app, omit the `display` member from your web app manifest, or set its value to `browser`:
+
+```json
+{
+  "name": "PWAmp music player",
+  "lang": "en-US",
+  "start_url": "/",
+  "short_name": "PWAmp",
+  "description": "A skinable music player app to play your favorite mp3 files",
+  "display": "browser",
+  "icons": [
+    {
+      "src": "./favicon-256.png",
+      "sizes": "256x256"
+    }
+  ],
+  "edge_side_panel": {
+    "preferred_width": 480
+  }
+}
+```
+
+
+<!-- ====================================================================== -->
+## Detect when your app runs in the sidebar
+
+Detecting when your app runs in the sidebar can be useful to provide your users with the best possible experience when your app runs in this specific browser UI surface.
+
+You can detect when your app runs in the sidebar by using User-Agent Client Hints. To learn more about User-Agent Client Hints, see [Detecting Microsoft Edge from your website](../../web-platform/user-agent-guidance.md).
+
+To detect when your app runs in the sidebar, do one of the following:
+
+* On your web server, read the `Sec-CH-UA` HTTPS request header and look for the `Edge Side Panel` brand. For example:
```

Comment:
As Hoch mentioned, client hint headers will not be in 112, but planned to be upcoming soon (Target 113). We could say, coming soon/next?

Reply:
Since we're waiting for feedback on the explainer anyway, maybe by the time we're ready to land the docs, the new CH headers will be available.
If they're not, I'll remove this part of the docs. It's better to document only what's available than to say some other pieces will come soon.

Reply:
This is being enabled through `msAddEdgeSidePaneBrandUA` feature flag.